llmfoundry/utils/builders.py

```diff
@@ -147,7 +147,7 @@ def build_optimizer(model: torch.nn.Module, name: str,
         return DecoupledClipLion(model.parameters(), **optimizer_config)
     elif name == 'adalr_lion':
         return DecoupledAdaLRLion(model.parameters(), **optimizer_config)
-    elif name == 'decoupled_lionw_8b':
+    elif name == 'decoupled_simbaw':
```

Comment:
this would be an API breaking change, so we would need to support both names here